### PR TITLE
optimize some serial e2e cases

### PR DESCRIPTION
**Pull request by KanShiori**
### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [x] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -456,93 +455,77 @@ func (oa *OperatorActions) CleanCRDOrDie() {
 	}
 }
 
-func (oa *OperatorActions) CreateOrReplaceCRD(isCRDV1Supported bool) {
```
Comment (KanShiori) [inline tests/actions.go:459]: change to use simple method to install CRD, this function is too complex

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**